opdracht.js

```diff
@@ -0,0 +1,141 @@
+// OPDRACHTEN
+
+// Opdracht 1b
+
+const soldOut = inventory.filter((tv) => {
+    return tv.sold === tv.originalStock;
+})
+
+console.log(soldOut);
+
+// Opdracht 1c
+
+const hasAmbiLight = inventory.filter((tv) => {
+    return tv.options.ambiLight === true;
+})
```

Comment:
Dit zou je nog wat korter kunnen schrijven
```suggestion
const hasAmbiLight = inventory.filter(tv => tv.options.ambiLight);
```